ouroboros-consensus/src/Ouroboros/Storage/VolatileDB/Util.hs

```diff
@@ -93,15 +91,16 @@ filePath fd = "blocks-" ++ show fd ++ ".dat"
 sizeAndId :: (BlockSize, BlockInfo blockId) -> (BlockSize, blockId)
 sizeAndId (size, bInfo) = (size, bbid bInfo)
 
-tryVolDB :: forall m blockId a
-         .  (MonadCatch m, Show blockId, Typeable blockId)
-         => m a -> m (Either (VolatileDBError blockId) a)
-tryVolDB = fmap squash . try . try
+tryVolDB :: forall m blockId a. (Monad m, Show blockId)
+         => ErrorHandling FsError                   m
+         -> ErrorHandling (VolatileDBError blockId) m
+         -> m a -> m (Either (VolatileDBError blockId) a)
+tryVolDB fsErr volDBErr = fmap squash . EH.try fsErr . EH.try volDBErr
```

Comment:
Ok, same pattern as `tryImmDb`. Looks fine. Want to add a similar haddock comment to explain what its purpose is?